<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>494</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>351</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tasks_Tab">
     <attribute name="title">
      <string>Tasks</string>
     </attribute>
     <widget class="QLineEdit" name="addtask_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addtask_pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/plus.png</normaloff>Icons/plus.png</iconset>
      </property>
     </widget>
     <widget class="QDateEdit" name="deadline_dateEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="deadlineDate_CheckBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTimeEdit" name="deadline_timeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="deadlineTime_CheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tasks_tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>331</width>
        <height>331</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>109</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Left</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deadline</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="deletetask_pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icons/sticky-note--minus.png</normaloff>Icons/sticky-note--minus.png</iconset>
      </property>
     </widget>
     <widget class="QComboBox" name="timesort_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>78</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Today</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Up to 7 days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Up to 14 days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delayed</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="completed_Tab">
     <attribute name="title">
      <string>Completed</string>
     </attribute>
     <widget class="QTableWidget" name="completed_tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>346</width>
        <height>441</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>109</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Left</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deadline</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="refreshtask_pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/arrow-circle-315-left.png</normaloff>Icons/arrow-circle-315-left.png</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leave_pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/door-open-out.png</normaloff>Icons/door-open-out.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/database--plus.png</normaloff>Icons/database--plus.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
